scoreboard_player_runes.py

```diff
@@ -0,0 +1,41 @@
+import re
+import json
+
+from mwcleric.fandom_client import FandomClient
+from mwcleric.auth_credentials import AuthCredentials
+from mwcleric.template_modifier import TemplateModifierBase
+from mwparserfromhell.nodes import Template
+from mwparserfromhell.nodes.extras.parameter import Parameter
+
+from runes_reforged_parser import RunesReforgedParser
+
+rune_parser = RunesReforgedParser()
+
+class TemplateModifier(TemplateModifierBase):
+    def update_template(self, template: Template):
+        for param in template.params:
+            param: Parameter
+            if param.name.matches('runes'):
+                w = param.value
+                for tl in w.filter_templates():
+                    tl: Template
+                    runes = tl.params[0]
```

Comment:
similar to the above, I would say it's safer to do `runes = tl.get('1')` - this explicitly gets the anonymous first param

you're also gonna need a `.value.strip()` here

Reply:
Also fixed